raising-jiras.md

```diff
@@ -0,0 +1,87 @@
+#Raising Jiras
+
+<p>Jiras are used at Holiday Extras as a means for teams to communicate what work is required (including data and insights). This doc seeks to outline some general guidelines and best practice when writing a jira.</p>
+
+##Title:
+<p>This is a brief description of the problem to be solved. Be clear and to the point.</p>
+
+**Good example:**
+<p>"Change the endpoints in HAPI to new Horizon service for Flexible Car Hire"</p>
+
+**Bad example:**
+<p>"Barbados car hire GM"</p>
+
+##Description:
+<p>This will vary depending on the type of ticket being raised. Below are some templates that can be used for different types of work (recognising that there isn't a one size fits all). The aim with using templates is to help collate all the information up front, so meeting the standards of the Definition of Ready, and to ensure consistency in approach across the business.</p>
+
+- Developers
+- Insights
+- Testing
+- Requests from the business
+<p>*(Will add in links to headings on 2nd iteration)*</p>
+
+###Developers
+
+**1. Requirements**
+<p>This is the 'what', outlining the details of the work required or the problem to be solved. Some key things to consider:</p>
```

Comment:
Could reverse that sentence - start with problem to be solved followed by details of the work. 

At what point does the 'how' become clear? And does this ever get recorded on the jira? Should it?